crates/ndc-query-plan/src/plan_for_query_request/mod.rs

```diff
@@ -137,10 +137,16 @@ fn plan_for_aggregate<T: QueryContext>(
     aggregate: ndc::Aggregate,
 ) -> Result<plan::Aggregate<T>> {
     match aggregate {
-        ndc::Aggregate::ColumnCount { column, distinct } => {
-            Ok(plan::Aggregate::ColumnCount { column, distinct })
-        }
-        ndc::Aggregate::SingleColumn { column, function } => {
+        ndc::Aggregate::ColumnCount {
+            column,
+            distinct,
+            field_path: _,
+        } => Ok(plan::Aggregate::ColumnCount { column, distinct }),
+        ndc::Aggregate::SingleColumn {
+            column,
+            function,
+            field_path: _,
+        } => {
```

Comment:
We want to forward field paths here. But we need to do a pass of testing and cleaning up aggregate support anyway so that can wait a bit.